<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviesAddModifyForm</class>
 <widget class="QDialog" name="MoviesAddModifyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>635</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MyResources.qrc">
    <normaloff>:/Images/movie pic.png</normaloff>:/Images/movie pic.png</iconset>
  </property>
  <widget class="QPushButton" name="pbtOk">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>592</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Release date *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledTrailer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Duration *</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledTitle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Country *</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblIdlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>590</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Title *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Age restriction *</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Trailer</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblId">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxCountry">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dedReleaseDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxAgeRestriction">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Poster</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledPoster">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Showing start date *</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dedShowingStartDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Showing end date *</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dedShowingEndDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Genres *</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwdGenres">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>161</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="tdtSynopsis">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>311</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>330</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Synopsis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtTrailer">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtPoster">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbDuration">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="MyResources.qrc"/>
 </resources>
 <connections/>
</ui>
